app/livechat/server/lib/Livechat.js

```diff
@@ -1229,25 +1229,13 @@ export const Livechat = {
 
 			const datetime = moment.tz(message.ts, timezone).locale(userLanguage).format('LLL');
 			const singleMessage = `
-				<p><strong>${author}</strong>  <em>${datetime}</em></p>
-				<p>${message.msg}</p>
+				${author}  ${datetime}\n
+				${message.msg}\n
 			`;
-			html += singleMessage;
+			text += singleMessage;
```

Comment:
@AlexanderKanakis Have you pushed the changes for updating the format of the text content?